pyspecProcScripts/third_level/process_enhancement.py

```diff
@@ -18,22 +24,19 @@
 logger = init_logging("info")
 t2 = symbols('t2')
 thesecolors = cycle(list('bgrcmykw'))
-def select_pathway(s,pathway):
-    retval = s
-    for k,v in pathway.items():
-        retval = retval[k,v]
-    return retval
```

Comment:
this is imported from elsewhere now?

Reply:
I didn't even notice this was deleted good catch!